test/Pantry-test.js

```diff
@@ -33,15 +33,75 @@ describe('Pantry', () => {
   });
   it('should be able to accept a recipe as an argument', () => {
     const recipe = new Recipe();
-    const pantry = new Pantry();
+    const pantry = new Pantry(recipe);
     const result = pantry.checkPantry(recipe);
 
     expect(result).to.equal(recipe);
   });
```

Comment:
I think I'm a little confused by this one, does this say that checkpantry is returning the recipe? does this test pass while the next one does?